features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/utils/DrawableUtils.kt

```diff
@@ -114,7 +115,20 @@ internal class DrawableUtils(
             canvas.drawColor(Color.TRANSPARENT, PorterDuff.Mode.MULTIPLY)
 
             drawable.setBounds(0, 0, canvas.width, canvas.height)
-            drawable.draw(canvas)
+
+            try {
+                drawable.draw(canvas)
+            } catch (exception: AndroidRuntimeException) {
+                internalLogger.log(
+                    InternalLogger.Level.ERROR,
+                    listOf(InternalLogger.Target.TELEMETRY),
```

Comment:
very nit: there is an overload which takes a single target

```suggestion
                    InternalLogger.Target.TELEMETRY,
```

Reply:
fixed